network.nix

```diff
@@ -1,6 +1,6 @@
 {
   quadletUtils,
-  pkgs,
+  podman,
```

Comment:
nit: maybe we can put the package into `quadletUtils` so we can have consistent interface across container, network, etc.

Reply:
Done